build/quickstarts-showcase/src/main/assembly/scripts/runQuickstarts.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Change directory to the directory of the script
+cd "$(dirname $0)" || exit
+
+jvmOptions="-Xms128m -Xmx512m -Dstartup-open-browser=true"
+mainJar="quickstarts/binaries/quarkus-app/quarkus-run.jar"
+
+echo "Usage: ./runQuickstarts.sh"
+echo "Notes:"
+echo "- Java 11 or higher must be installed. Get the latest OpenJDK from (https://adoptium.net/)."
```

Comment:
We're minimum java 17

Reply:
The way we currently run the showcase, we don't even need the script. I will remove it.